lib/chttp/http.go

```diff
@@ -58,7 +58,7 @@ func prettyPrintJSON(b []byte) string {
 	for i, p := range parts {
 		if b := []byte(p); json.Valid(b) {
 			var out bytes.Buffer
-			// nolint:errcheck
+			//nolint:errcheck
```

Comment:
lib/chttp/http.go:61:4: directive `// nolint:errcheck` should be written without leading space as `//nolint:errcheck` (nolintlint)
```
                        // nolint:errcheck
                        ^
```